saos-restful-web-api/src/main/resources/message/restfulapi.properties

```diff
@@ -2,5 +2,6 @@ dump.entry.point = Entry point for dump services, use it if you want to download
 search.entry.point = Entry point for search services, use it if you want to build UI application using our system as backend
 dump.courts.description = Allows download information about common courts
 dump.judgments.description = Allows download information about judgments
+dump.judgments.enrichments.description = Allows download information about enrichments of judgments
 dump.supreme.court.chambers.description = Allows download information about supreme court chambers
```

Comment:
Krzysiek, change please this 'Allows' sentences into some more appropriate ones, for example:
`dump.judgments.description = Judgment data dump download`